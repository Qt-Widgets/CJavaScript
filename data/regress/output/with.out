314.15926535897933
-10
10
