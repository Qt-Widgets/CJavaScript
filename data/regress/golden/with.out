314.1592653589793
-10
10
